README.md

```diff
@@ -483,11 +512,13 @@ result of undefined behavior that also scrambled memory or causes
 
 ## Development
 
-The usual procedures for development with CMake apply.  One typical
+The usual procedures for development with CMake apply. One typical
 set of commands might be:
 
 ```sh
 cmake -DBUILD_TESTING=1 -Wno-dev -S . -B ../build -GNinja
 cmake --build ../build
 ctest --output-on-failure --test-dir ../build
 ```
+
+**VSCode Note**: The CMake Test Explorer integration is disabled in the `.vscode/settings.json` file. It breaks the test detection. To run tests use the `Run CTest` command. If you toggle the integration option on and run the tests and then toggle it off again and refresh the Test Explorer panel, you can then use the panel to run and debug tests individually. If you see `_NOT_BUILT` in the ctest output, it is because test explorer integration is enabled. The issue is reported [here](https://github.com/microsoft/vscode-cmake-tools/issues/3358#issuecomment-2403267565).
```

Comment:
```suggestion
**VSCode CMake Plugin Note**: Test Explorer integration has been disabled in the `.vscode/settings.json` file, because it [breaks test detection](https://github.com/microsoft/vscode-cmake-tools/issues/3358). If you see `_NOT_BUILT` in the CTest output, test explorer integration was somehow enabled at configure time. To run tests use the `Run CTest` command. If you toggle the integration option on and run the tests and then toggle it off again and refresh the Test Explorer panel, you can then use the panel to run and debug tests individually. 
```

> To run tests use the `Run CTest` command

Is this something we need to say?  All the ways of running the tests seem to work when I do it.  Nothing wrong with clicking on one of the several icons…

> you can then use the panel to run and debug tests individually

Do we really want to promise that this hack will work?  I don't really understand how to control this.  I can delete the cache and reconfigure and integration still works, but if I delete the build directory, of course, it breaks again.  What happens if you add a new test?  Does it work or not?  I don't trust this.

Reply:
Any thoughts about my questions?  My suggested edits didn't account for them, because they are questions.

Reply:
?